dist/s_string.ok

```diff
@@ -64,6 +63,7 @@ CPP
 CPUs
 CRC
 CSV
+CSlmvxz
```

Comment:
This change is one of the issues in WT-11228. I don't think the changes in that ticket are large. If you want to fold them into this PR that would be fine and then close that as a dup or gone away.

Reply:
If that's ok I will keep [WT-11228](https://jira.mongodb.org/browse/WT-11228) separate as it will impact other files and I think we can use that ticket to improve the way we deal with common arguments and test-level arguments.